contributing.md

```diff
@@ -0,0 +1,17 @@
+# Sanoma Learning Design System contributing guide
+
+We would like to welcome all contributors.
+
+These are the basic contributing rules:
+- New branch should be created from the `main` branch
+- Branch naming: `feature/[#issueNumber]-...-...` or `fix/[#issueNumber]-...-...`, eg `feature/[#21]-button-component`
+- Please use English for naming 🙂
+- Every component should have prefix `sl-` eg. `sl-button`
+- Please add unit tests for your changes
+- Commit message with scope (eg. core, storybook, website, tokens) for instance:
+    `(core): add button component`
+- Before creating PR – the newest version of `main` branch should be merged into feature branch and all conflicts should be resolved
+- Squash before merge
+- Pull request naming - eg `Task_number – (scope): description` ->
```

Comment:
I don't think this is needed: changset will do this for you as well:
<img width="423" alt="Screenshot 2022-11-29 at 12 25 16" src="https://user-images.githubusercontent.com/3968/204516953-18a7b2b0-0270-4bd6-b823-83da5aa345ac.png">